strings/strings.c

```diff
@@ -20,7 +24,18 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+    int counter=string_length(s)-1;
+    int rv_count=0;
+    while (counter>=0) {
+        rv[rv_count]=s[counter];
+        if (s[counter]==' '){
```

Comment:
curious what this if statement is doing.  string still reverses correctly without it.

Reply:
wouldn't work on repl unless I did that tried removing now and it works on terminal